distribution/lib/Standard/Database/0.0.0-dev/src/Connection/Postgres_Details.enso

```diff
@@ -27,6 +27,7 @@ type Postgres_Details
          No Authentication or the PGPass file).
        - use_ssl: Whether to use SSL (defaults to `SSL_Mode.Prefer`).
        - client_cert: The client certificate to use or `Nothing` if not needed.
+    @credentials Credentials.default_widget
```

Comment:
I tried overriding the widget to get the Credentials selector to display. Unfortunately this did not seem to work. Open to suggestions how this could be fixed.